VideoLocker/res/values/styles.xml

```diff
@@ -106,6 +105,10 @@
         <item name="android:background">@color/edx_brand_primary_base</item>
     </style>
 
+    <style name="course_dashboard_separator_style" parent="separator_style">
```

Comment:
we have a total of 3 separator color-types:
1. grey
2. red
3. blue
So, i guess we can create 3 children and 1 parent styles to cater them all.